src/hero.tsx

```diff
@@ -24,13 +25,17 @@ import type {DataAttributes, RendersElement, RendersNullableElement} from './uti
 
 type LayoutProps = {children: React.ReactNode; isInverse: boolean};
 
-// This is a duplication of the ResponsiveLayout, needed because original ResponsiveLayout has a inner div that we cannot control and the height is missing there.
-// We need to control this height to fix the actions at the bottom of the Hero component when we are inside a Slidehow with differents Hero heights.
 const Layout = ({children, isInverse}: LayoutProps) => {
     return (
-        <ThemeVariant isInverse={isInverse}>
-            <div className={styles.layout}>{children}</div>
-        </ThemeVariant>
+        <InternalResponsiveLayout
+            isInverse={isInverse}
+            className={styles.layout}
+            innerDivClassName={styles.layout}
+            shouldExpandWhenNested="desktop"
```

Comment:
only in desktop? why?

Reply:
In mobile, the picture is rendered outside the responsive layout and the content has some margin relative to the picture because of this responsive layout (expected and intended).

Without expanding (now):

![Screenshot from 2024-04-12 16-28-28](https://github.com/Telefonica/mistica-web/assets/25785151/69c7ca2a-9714-4d42-af24-afc436228631)

If we expand it (not correct):
![Screenshot from 2024-04-12 16-29-17](https://github.com/Telefonica/mistica-web/assets/25785151/a28e1289-27e5-4f9d-bd67-896d8e4b8664)

Reply:
I think you're wrong. See:

https://mistica-m3jn5uk7h-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwAIyALCPAYVIjQGUxqYBbGFmEmMABdmAPgA6AOwoUkACRhoIk6dIBGAQzABrAOaKArhKgBeMSAkQJMM8pUWU6qFGwSdATVvTuz9ceCeVJABJTnUdaykVKLI0MFMQakFBAAcyeAB6dLIIfViYADpDMmSSdRp8yE50gEYANgAGeoAPAE5G9IB%2BADNciWxBXOsQAKiy5IFBACV1QWwIeLr4FptIqPTxVYoAXwCORxIXGD8kAjCKQQBPcfjNWcwhkXl9w6R0050RHc3kjFnBfni6Bgf34KyiIKO-hABH6oJAXyisDIYDQ2GSs0s8QI1GwlFxFHUFBIlh0FCRKLRGKkeH61AJRJJ5xgTUE5wgVBgMAotDw5xxlDwzC0ZDBKmZgjQvmAqFKYBgtBIsDQFHWCJUqn0SUx0oAQprhBJ0NhQmgLhQuuotDARABBIRzCSvPVaw2ok0XT4BDUugAyLi0x2dBr9Ei03IwXXiAGIzCIQ1onfrLPHPZsoDN1DakqjvTAyH5gEyyLMoPAKGYOIozNs1dIkVphMkALIwHwoCBkfoO%2BKonSJUXrWxISZ55KWTv3H3qC45VmYdSo9QSQTxFz3NBkB4BOQKJSbNSaXQGIzxCxWUVReyOZyuDz7ijebBSkaBEJhCJRT9UWLxRIpNKZNkuRyoUEjFKU5SVDUDTNG09SdD0aB9AMGAXl%2BBLFBM0wYgstRLGhawbF%2BtYqHsUAHFYxzvOcVxHGYtzYPcsZPORLxvGEqZfj8wKwnRIBAhCBHnLxfhmDC-xDCRdZ5hS6LdmJ-IUPihLEq4ZIyaicmWBQNKCHSKmMoI4pshyXI8ny%2BKCmgwpCeKkrHCgsryqQSoqpxn7egagZJq6xoLmaFpWra9qWImLpGu67lRJ5yb%2Bt5vr%2BuGMCRmYMYgHG-phcG-pRSo6aCJm2bYLm%2BbAIWRnFtgpblgku7VlsUnqWQDYQM2rZPu2nZUj22B9oIaGDpErwjsU46MTAU4zpqGyvECOD4EQpBkBsIAADQgHpXB5ggADaIA6qU%2BggAAuutNJQHpZC7QAzA0q0AOy1AAHKt1T1AATAALMdWxAA

There are 2 Heros in this playroom. One inside a ResponsiveLayout and one outside it, the one inside the it should expand and be aligned as the one outside

Reply:
It should be fixed now :)